src/Serilog.Sinks.Skywalking/README.md

```diff
@@ -0,0 +1,49 @@
+# serilog-sinks-skywalking
+[![Nuget](https://img.shields.io/nuget/v/Serilog.Sinks.Skywalking)](https://www.nuget.org/packages/Serilog.Sinks.Skywalking/)
```

Comment:
https://www.nuget.org/packages/Serilog.Sinks.Skywalking/ This is tricky. We can't host a project controlled by you.
There are two reasons
1. In SkyAPM-dotnet agent maintainer perspective, all codes hosted here should be released on this maintainer team under skyapm umbrella. 
2. We don't want to set too strict rules, but as several maintainers of this repo are also PMC members of Apache SkyWalking, we chose skyapm-xxx to avoid brand issues with an Apache Software Foundation Branding.

So, from maintenance and branding perspective, we can't and shouldn't use this.

If this would be accepted as a component by @liuhaoyang @snakorse , this component should be like `SkyAPM-dotnet` component integrated into a component like `SkyAPM-dotnet-serilog-sink` or similar.

Then it could be released with this repository's other components under the same org.